Makefile

```diff
@@ -27,7 +27,7 @@ OCTAVE ?= octave
 MKOCTFILE ?= mkoctfile -Wall
 MATLAB ?= matlab
 
-TEST_CODE=success = doctest({'doctest', 'test/', 'test/examples/'}); exit(~success);
+TEST_CODE=ver('octave'), success = doctest({'doctest', 'test/', 'test/examples/'}); exit(~success);
```

Comment:
How about just `ver`? `TEST_CODE` is shared with `matlab_test`. In Matlab this warns about an unknown toolbox.